### PR TITLE
Fix docs of gcs offload

### PR DESCRIPTION
**Pull request by wolfstudy**
Signed-off-by: xiaolong.ran <rxl@apache.org>

### Modifications

- Fix docs of gcs offload

### CODE REVIEW

File: site2/docs/cookbooks-tiered-storage.md [hunk 1]
```diff
@@ -168,12 +168,15 @@ a Json file, containing the GCS credentials of a service account.
 more information of how to create this key file for authentication. More information about google cloud IAM
 is available [here](https://cloud.google.com/storage/docs/access-control/iam).
 
-Usually these are the steps to create the authentication file:
-1. Open the API Console Credentials page.
-2. If it's not already selected, select the project that you're creating credentials for.
-3. To set up a new service account, click New credentials and then select Service account key.
-4. Choose the service account to use for the key.
-5. Download the service account's public/private key as a JSON file that can be loaded by a Google API client library.
+To generate service-account credentials, or to view the public credentials that you've already generated, do the following:
```
Comment (Anonymitaet) [inline site2/docs/cookbooks-tiered-storage.md:171]:
```suggestion
To generate service account credentials or view the public credentials that you've already generated, follow the following steps:
```

File: site2/docs/cookbooks-tiered-storage.md [hunk 2]
```diff
@@ -168,12 +168,15 @@ a Json file, containing the GCS credentials of a service account.
...
+
+1. Open the [Service accounts page](https://console.developers.google.com/iam-admin/serviceaccounts).
+2. If prompted, select a project, or create a new one.
```
Comment (Anonymitaet) [inline site2/docs/cookbooks-tiered-storage.md:174]:
```suggestion
2. Select a project or create a new one.
```

File: site2/docs/cookbooks-tiered-storage.md [hunk 3]
```diff
@@ -168,12 +168,15 @@ a Json file, containing the GCS credentials of a service account.
...
+3. Click **Create service account**.
+4. In the **Create service account** window, type a name for the service account, and select **Furnish a new private key**. If you want to [grant G Suite domain-wide authority](https://developers.google.com/identity/protocols/OAuth2ServiceAccount#delegatingauthority) to the service account, also select **Enable G Suite Domain-wide Delegation**.
+5. Click **Create**.
+
+> Notes: Make ensure that the service account you create has permission to operate gcs, please assign **Storage Admin** permission to your service account in [here](https://cloud.google.com/storage/docs/access-control/iam)
```
Comment (Anonymitaet) [inline site2/docs/cookbooks-tiered-storage.md:179]:
```suggestion
> Notes: Make ensure that the service account you create has permission to operate GCS, so you need to assign **Storage Admin** permission to your service account [here](https://cloud.google.com/storage/docs/access-control/iam).
```

**Comment by wolfstudy** [comment]
@Anonymitaet fixed done, PTAL again

**Pull request merged**